docs/source/user_guide/model_understanding.ipynb

```diff
@@ -38,11 +38,51 @@
    "source": [
     "import evalml\n",
     "\n",
-    "class RFBinaryClassificationPipeline(evalml.pipelines.BinaryClassificationPipeline):\n",
-    "    component_graph = ['Simple Imputer', 'Random Forest Classifier']\n",
+    "class DTBinaryClassificationPipeline(evalml.pipelines.BinaryClassificationPipeline):\n",
+    "    component_graph = ['Simple Imputer', 'Decision Tree Classifier']\n",
     "\n",
     "X, y = evalml.demos.load_breast_cancer()\n",
     "\n",
+    "pipeline_dt = DTBinaryClassificationPipeline({})\n",
+    "pipeline_dt.fit(X, y)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Tree Visualization\n",
+    "\n",
+    "We can visualize the structure of the Decision Tree that was fit to that data, and save it if necessary."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from evalml.model_understanding.graphs import visualize_decision_tree\n",
+    "\n",
+    "visualize_decision_tree(pipeline_dt.estimator, max_depth=2, rotate=False, filled=True, filepath=None)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's make the pipeline more complex by replacing the Tree with a Forest."
```

Comment:
Instead of replacing the entire pipeline, which would have resulted in reduced performance and some uglier graphs, I chose to start with a DecisionTree and then change it